annotation_tracker/web_client/panels/Experiments/index.js

```diff
@@ -32,8 +32,24 @@ const Experiments = Panel.extend({
             'input': false
         }; // Utilized for toggling the different sections in the experiments panel
         this.sessionStarted = false;
+        this.resizeObservers = []; // [observer, element]
+    },
+    observePanels() {
+        $('.s-panel').each((_index, element) => {
+            const observer = new ResizeObserver((data) => {
+                activityLogger.logPanelPositions();        
+            });
+            observer.observe(element);
+            this.resizeObservers.push([observer, element]);
+        });
+    },
+    unobservePanels() {
+        if (this.resizeObservers.length > 0) {
+            this.resizeObservers.forEach(([observer, element]) => {
+                observer.unobserve(element);
+            })
+        }
```

Comment:
if you start and stop sessions a lot you would continually be adding to the `this.resizeObservers` array and double watching panels.